client/my-sites/stats/site.jsx

```diff
@@ -194,6 +197,7 @@ export default connect(
 	state => {
 		const siteId = getSelectedSiteId( state );
```

Comment:
Considering your comment below https://github.com/Automattic/wp-calypso/pull/13021/files#r112488420, this is also a good point. Thanks